.konflux/rpms.lock.yaml

```diff
@@ -0,0 +1,326 @@
+# This file was generated via the following command:
```

Comment:
Not something missing from this PR specifically--just an open question: What is our process going to be for keeping this up-to-date?

Reply:
This is supported by Renovate/Mintmaker, so we'll be getting updates via automated PRs.

Reply:
So we will need to move the file out of the nicely organized `.konflux` folder =/